src/containers/Interfaces/AlterForm.js

```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import { bindActionCreators, compose } from 'redux';
+import { connect } from 'react-redux';
+import { isValid, isSubmitting, submit } from 'redux-form';
+import Swiper from 'react-id-swiper';
+import { Scroller, ProgressBar } from '../../components';
+import Node from '../Node';
+import { actionCreators as sessionsActions } from '../../ducks/modules/sessions';
+import { nodeAttributesProperty, nodePrimaryKeyProperty } from '../../ducks/modules/network';
+import { makeNetworkNodesForType } from '../../selectors/interface';
+import { protocolForms } from '../../selectors/protocol';
+
+import { Form } from '../';
+import { getItemComponent } from './Information';
+import { getCSSVariableAsNumber } from '../../ui/utils/CSSVariables';
+
+class AlterForm extends Component {
+  constructor(props) {
+    super(props);
+    this.swipeRef = React.createRef();
+    this.state = {
+      activeIndex: 0,
+    };
+  }
+
+  onSlideTransition = () => {
+    if (this.swipeRef.current && this.swipeRef.current.swiper) {
+      const submitIndex = this.swipeRef.current.swiper.previousIndex;
+      // try to submit (on fail, form is "touched" and errors can show)
+      this.props.submitForm(this.getNodeFormName(submitIndex));
+      if (!this.formSubmitAllowed(submitIndex)) {
```

Comment:
UX note: it felt unusual that I couldn't go *back* if the form didn't validate. By contrast, on the modal NodeForm, I can always back out by cancelling the modal if I'm not ready to fill the required fields for some reason. Here, I'd have to use the settings/timeline menu.

Reply:
I thought this, too. I think going back should always be allowed. Perhaps something like "if the navigation is backwards and the form is invalid, don't submit the form".